client/src/components/footer/footer.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './footer.css';
+export default function Footer() {
+  return (
+    <footer>
+      <div className='logo-container'>
+        <h2>Logo</h2>
+      </div>
+      <section className='footer-links'>
+        <article>
+          <h3>Links</h3>
+          <ul>
+            <a href='#'>Link</a>
+            <a href='#'>Link</a>
+            <a href='#'>Link</a>
+            <a href='#'>Link</a>
+          </ul>
```

Comment:
Hi @BryonVerdone. Good work here! A best practice for semantic and valid html is to use an li (list element) element as a direct child of the ul. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li. Let me know if you have any questions. 
```suggestion
          <ul>
            <li><a href='#'>Link</a></li>
            <li><a href='#'>Link</a></li>
            <li><a href='#'>Link</a></li>
            <li><a href='#'>Link</a></li>
          </ul>
```